dotnet/samples/Concepts/Agents/ChatCompletion_Mem0.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net.Http.Headers;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Memory;
+
+namespace Agents;
+
+#pragma warning disable SKEXP0130 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+
+/// <summary>
+/// Demonstrate creation of <see cref="ChatCompletionAgent"/> and
+/// adding memory capabilities to it using https://mem0.ai/.
+/// </summary>
+public class ChatCompletion_Mem0(ITestOutputHelper output) : BaseTest(output)
+{
+    private const string AgentName = "FriendlyAssistant";
+    private const string AgentInstructions = "You are a friendly assistant";
+
+    /// <summary>
+    /// Shows how to allow an agent to remember user preferences across multiple threads.
+    /// </summary>
+    [Fact]
+    private async Task UseMemoryAsync()
```

Comment:
Remove async, let the method be more descriptive.

```suggestion
    private async Task UseChatAgentWithMemoryAcrossThreads()
```